src/platform/meteorlake/include/platform/lib/memory.h

```diff
@@ -56,7 +56,7 @@
 /**
  * size of HPSRAM system heap
  */
-#define HEAPMEM_SIZE 0x40000
+#define HEAPMEM_SIZE 0xD0000
```

Comment:
I think it is good to provide more precise explanation in commit message why you're increasing HEAPMEM_SIZE by 0x90000, i.e. in case of audio format 16000Hz/6ch/24bit history_buffer_size = 16 * 6 * 3 * 2100s = 604800 bytes (0x93A80)